packages/react-sdk/src/stitches/stitches.ts

```diff
@@ -1,31 +1,30 @@
-import {
-  createStitches,
-  globalCss as globalCssImport,
-  type CSS,
-  css as createCss,
-} from "@stitches/core";
+import { createStitches, type CSS } from "@stitches/core";
 import type { Breakpoint } from "../css";
 
 let media = {};
 
-// @todo needs fixing
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-let stitches: any;
+const createStitchesInstance = () => createStitches({ media });
+
+type StitchesInstance = ReturnType<typeof createStitchesInstance>;
+
+let stitches: StitchesInstance | undefined;
 
 export { type CSS };
 
 const getCachedConfig = () => {
   if (stitches === undefined) {
-    stitches = createStitches({ media });
+    stitches = createStitchesInstance();
   }
   return stitches;
 };
 
-export const css: typeof createCss = (...args) => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const css: StitchesInstance["css"] = (...args: any[]) => {
```

Comment:
This is closer to the intention because we reexport a proxy to your own instance and not to some default instance embedded in the Stitches package.

`any[]` is needed here though because TS doesn't understand that this is merely a proxy function and while instantiating the generic call embedded in this function it fails to prove that this is OK.

Reply:
are we still geting any ts validation for css()?

Reply:
Yes, that flows from the declared type (`StitchesInstance['css']`). We only suppress the typechecking "internally" within this function - but externally this doesn't change anything.